libs/mutantschemata/cpp_source/type.hpp

```diff
@@ -17,12 +17,12 @@ C++ types for easier communication between C++ and D code.
 namespace CppType {
 
 struct SourceLoc {
-    unsigned line;
-    unsigned column;
+    unsigned int line;
+    unsigned int column;
 };
 struct Offset {
-    unsigned begin;
-    unsigned end;
+    unsigned int begin;
```

Comment:
Tip: use `uint64_t` from `cstdint` here to ensure that it is always a 64bit unsigned int.
Then on the D side use `ulong`.
Then it will work on all platforms (32 and 64 bit).

See [C and D data type compatibility](https://dlang.org/spec/interfaceToC.html#data_type_compat).